server/mdm/apple/apple_mdm.go

```diff
@@ -906,3 +906,59 @@ func IOSiPadOSRefetch(ctx context.Context, ds fleet.Datastore, commander *MDMApp
 	}
 	return nil
 }
+
+var otaMobileConfigTemplate = template.Must(template.New("").Parse(`<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
two notes:

1. could you move this template to the `mobileconfig` package? we've been trying to keep them all there
2. could you add `.Option("missingkey=error")`?

Reply:
if `1` it's not possible due to import cycles (this happens for the enrollment profile) please ignore